package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "@ulaval/modul-vuex",
+    "version": "1.0.1-1",
+    "description": "MODUL Vuex utilities",
+    "author": "Martin Simard <martin.simard@dti.ulaval.ca>",
+    "private": false,
+    "scripts": {
+        "clean": "rimraf dist",
+        "reset": "rmdir /S /Q node_modules",
```

Comment:
Why do we use rmdir and not rimraf ?

Reply:
"reset" is a script used internally (probably just myself!). It's a fast way to delete the node_modules folder when I switch from branch to branch, prepare release, etc. Since rimraf is a component that has to be installed in the node_modules, a Node component cannot delete the node_modules folder itself.